src/init.h

```diff
@@ -8,8 +8,11 @@
 
 #include <memory>
 #include <string>
+#include <vector>
 #include <util/system.h>
 
+static const std::vector<std::string> CHECKLEVEL_DOC{"level 0 reads the blocks from disk", "level 1 verifies block validity", "level 2 verifies undo data", "level 3 checks disconnection of tip blocks", "level 4 tries to reconnect the blocks", "each level includes the checks of the previous levels"};
```

Comment:
I think it would be slightly easier to read if each was on their own line (like before). 
```suggestion
static const std::vector<std::string> CHECKLEVEL_DOC{
"level 0 reads the blocks from disk",
...
"each level includes the checks of the previous levels",
};
```

Also, a better place to put this would be in validation.h (just the declaration) and then define it in validation.cpp.

`init.h` seem an odd place to put RPC documentation.

Reply:
I've changed it so that each is in their own line but wasn't quite sure how to just declare in the .h and define in .cpp without the compiler telling me I was re-declaring it.

At the moment  ```CHECKLEVEL_DOC``` is declared and defined in ```validation.h```